docs/reference/resp2-to-resp3-replies.md

```diff
@@ -0,0 +1,116 @@
+| SINTER <br/>SPOP COUNT<br/>SMEMBERS<br/>SRANDMEMBER COUNT<br/>SUNION | Array       | Set         |
+
+### Sorted Set
+| Command                  | RESP2 Reply                                                 | RESP3 Reply                                             |
+|--------------------------|-------------------------------------------------------------|---------------------------------------------------------|
+| ZADD INCR                | BlobString                                                  | Double                                                  |
+| ZDIFF WITHSCORES         | Array\<BlobString>                                          | Array<\[BlobString, Double]>                            |
+| ZINCRBY                  | BlobString                                                  | Double                                                  |
+| ZINTER WITHSCORES        | Array\<BlobString>                                          | Array<\[BlobString, Double]>                            |
+| ZMPOP                    | null &#124; \[BlobString, Array<\[BlobString, BlobString]>] | null &#124; \[BlobString, Array<\[BlobString, Double]>] |
+| ZMSCORE                  | Array\<null &#124; BlobString>                              | Array\<null &#124; Double>                              |
+| ZRANDMEMBER WITHSCORES   | Array\<BlobString>                                          | Array<\[BlobString, Double]>                            |
+| ZSCORE                   | BlobString                                                  | Double                                                  |
+| ZUNION WITHSCORES        | Array\<BlobString>                                          | Array<\[BlobString, Double]>                            |
+| ZRANK WITHSCORE          | \[number, BlobString]                                       | \[number, Double]                                       |
+| ZREVRANK WITHSCORE       | \[number, BlobString]                                       | \[number, Double]                                       |
+| ZRANGE WITHSCORES        | Array\<BlobString>                                          | Array<\[BlobString, Double]                             |
+| ZRANGEBYSCORE WITHSCORES | Array\<BlobString>                                          | Array<\[BlobString, Double]                             |
+| ZMPOP                    | \[BlobString, Array<\[BlobString, BlobString]>]             | \[BlobString, Array<\[BlobString, Double]>]             |
+| BZPOPMAX                 | null &#124; \[BlobString, BlobString, BlobString]           | null &#124; \[BlobString, BlobString, Double]           |
+| BZPOPMIN                 | null &#124; \[BlobString, BlobString, BlobString]           | null &#124; \[BlobString, BlobString, Double]           |
+| ZPOPMIN                  | null &#124; \[BlobString, BlobString]                       | null &#124; \[BlobString, Double]                       |
```

Comment:
sorry to be pedantic, but can we be super clear about what this syntax means? i.e. `Array<...>` vs `[...]` ? it *looks* like you're using `[x,y,z]` to mean an array of exactly these elements, and `Array<...>` to mean an array of some unknown quantity of elements; is that right? so `Array<\[x,y]>` is an array (unknown length) of jagged pairs?

Reply:
Added to the doc.